ci-operator/step-registry/addon/aws/osd/addon-aws-osd-workflow.yaml

```diff
@@ -0,0 +1,30 @@
+workflow:
+  as: addon-aws-osd
+  steps:
+    env:
+      API_HOST: stage
+      CHANNEL_GROUP: candidate
+      COMPUTE_MACHINE_TYPE: "m5.4xlarge"
+      MACHINE_POOL_INSTANCE_TYPE: "m5.2xlarge"
+      COMPUTE_NODES: "3"
+      PARALLEL: "true"
+      MULTI_AZ: "true"
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      MIN_REPLICAS: "3"
+      MAX_REPLICAS: "6"
+      ETCD_ENCRYPTION: "true"
+    pre:
+      - ref: osd-ccs-cluster-provision-aws
+      - ref: osd-ccs-conf-machine-pool
+      - ref: osd-ccs-conf-idp-htpasswd
+      - ref: osd-ccs-conf-idp-htpasswd-multi-users
+      - ref: osd-ccs-cluster-operators-wait-ready
+      - ref: addon-install
+    post:
+      - ref: osd-ccs-cluster-deprovision
+  documentation: |-
+    This workflow installs an OSD cluster with the specified configures using the lastest nightly build.
```

Comment:
not nightly but what is provided in OPENSHIFT_VERSION

Reply:
done